### PR TITLE
pokemon api

### PR DESCRIPTION
**Pull request by gr8rithic**
displays pokemon image based on the pokemon name given as input

### **Describe your change:**

* [ ] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [x] Documentation change?

### **Checklist:**
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [ ] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [x] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [ ] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

**Comment by gr8rithic** [comment]
Hello I am new to open source it will be good if I get a good guideline of what mistakes I am Making. Thank you very much

File: web_programming/pokemon_api.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+import matplotlib.pyplot as plt
+
+if __name__ == "__main__":
+    name = input("Enter the name of the Pokemon")
+    # The image URL is in the content field of the first meta tag
+    url = f"https://api.pokemontcg.io/v1/cards?name={name}"
+    print(url)
+    response = requests.get(url)
+    json = response.json()
+    recieved_data=json(name)
```
Comment (cclauss) [inline web_programming/pokemon_api.py:12]: I do not understand.  `json` is a library on line 2 and is data on line 11 and is a function on line 12.

Comment (gr8rithic) [reply web_programming/pokemon_api.py]: I have edited the file

File: web_programming/pokemon_api.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import requests
+import json
```
Comment (cclauss) [inline web_programming/pokemon_api.py:2]:
```suggestion
```

File: web_programming/pokemon_api.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+import matplotlib.pyplot as plt
+
```
Comment (cclauss) [inline web_programming/pokemon_api.py:4]:
```suggestion


```

File: web_programming/pokemon_api.py [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+if __name__ == "__main__":
+    name = input("Enter the name of the Pokemon")
+    # The image URL is in the content field of the first meta tag
+    url = f"https://api.pokemontcg.io/v1/cards?name={name}"
+    print(url)
+    response = requests.get(url)
+    recieved_data = response.json()
+    print(recieved_data)
+    url_data=requests.get(recieved_data["cards"][1]["imageUrl"])
+    with open("001.png","wb") as f:
```
Comment (cclauss) [inline web_programming/pokemon_api.py:14]:
```suggestion
    with open("001.png", "wb") as f:
```

**Pull request closed without merging**